client/src/modules/Mentor/components/StudentsTable/StudentsTable.tsx

```diff
@@ -0,0 +1,29 @@
+import { Col, Row, Table } from 'antd';
+import React from 'react';
+import { getColumns, StudentsTableProps, StudentsTableRow } from '.';
+
+function StudentsTable({ data, course }: StudentsTableProps) {
+  const getUniqueKey = (record: StudentsTableRow) => Object.values(record).filter(Boolean).join('|');
```

Comment:
I guess can be moved out of component scope

Reply:
Good point, thanks. Will do